boards/arm/nrf52820dongle_nrf52820/nrf52820dongle_nrf52820.dts

```diff
@@ -57,6 +57,8 @@
 		led1-red   = &led1_red;
 		led1-green = &led1_green;
 		led1-blue  = &led1_blue;
```

Comment:
Fix commit message title `MCUbootr's/MCUboot's`.
Also shouldn't it be `boards: arm: ...`